app/scaffold/scaffold.js

```diff
@@ -57,23 +58,27 @@ class Scaffold {
 
         model.restore();
 
-        // There is currently a bug in Safari 11.1 that causes this app to break. If a user is on Safari 11.1 display an error.
-
-        Utils.checkBrowser(DOM.alertMessageToScreenAndConsole, Message.notInitialized);
-
         // Listen
 
         this.addListeners();
 
+        // There is currently a bug in Safari 11.1 that causes this app to break. If a user is on Safari 11.1 display an error.
```

Comment:
Is this still true? Are we checking this in our tests?

Reply:
Yes, there is a bug with the complete version

Reply:
Ok